infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -905,7 +905,8 @@ private void recordLockRelease() {
 
     private void updateReplicationStatus() {
         if (!statusFlag) {
-            replicationManager.updateStatusAsNotStarted();
+            // Add default entry to Replication Status Table
+            replicationManager.initializeStatusAsNotStarted();
```

Comment:
there can be a delay between the time of startup and leadership acquisition.  Adding the initialization here does not eliminate that window.